lib/Twig/Extension/Core.php

```diff
@@ -408,7 +408,7 @@ function twig_date_converter(Twig_Environment $env, $date = null, $timezone = nu
         return false !== $timezone ? $date->setTimezone($timezone) : $date;
     }
 
-    if ($date instanceof DateTime || $date instanceof DateTimeInterface) {
```

Comment:
note to self, cannot be backported to 1.x line, `DateTimeInterface` introduced in 5.5